iterating-over-data/practice-iterating-over-data.md

```diff
@@ -0,0 +1,465 @@
+# Practice: Iterating Over Data
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: EJew-RW0t
+* title: Which Range is Correct?
+
+##### !question
+
+Given a function called `stopwatch` that takes in the argument `22`, which of the following loops would  correctly start at `1` and increment all the way up to *and include* `22`?
```

Comment:
We technically don't teach `range()`, but even if we did, this question assesses knowledge about `range()`, which seems a little out of place here? I'd be open for pushback because sometimes reinforcement like this is good, but the fact that it's both not-taught and out of place makes me want to remove it.

Reply:
I was wrong, apparently we DO teach range. Huh. LOOL